tests/test_target/test_class_balance.py

```diff
@@ -213,6 +213,9 @@ def test_pandas_occupancy_compare(self):
         # oz.finalize()
         self.assert_images_similar(oz, tol=0.5)  # w/o tol fails with RMS 0.433
 
+    @pytest.mark.xfail(
+        IS_WINDOWS_OR_CONDA,
+        reason="test stopped working on Miniconda, hangs indefinitely. See #1024")
```

Comment:
@bbengfort weirdly it seems like this is the test that is causing the problems. No idea why that would be